src/release/issue-triaging.md

```diff
@@ -0,0 +1,80 @@
+- Does this issue require incomplete or internal features? Add `requires-{incomplete,internal}-features`.
+
+For applying and removing labels, unprivileged users can use rustbot.
+For example, `@rustbot label +T-compiler +C-bug +A-linkage +O-macos -needs-triage`.
+
+To see a list of all labels, check out the "labels" page next to the search bar in the issue tracker.
+
+## Further triaging
+
+For issues that have been through the initial triaging step (that is, don't have the `needs-triage` label anymore), there are usually
+still things that can be improved. There are often many more labels that could be applied (using rustbot again if you don't have privileges).
+
+Additionally, old (there is no clear definition of old yet, but something on the order of months) `S-needs-repro` issues can be closed
+if there is no way to make progress without a reproduction. This requires privileges, but if you don't have them, you can just link the issue
+on Zulip (for example in `t-release/triage` or `general`) and someone with privileges can close it for you.
+
+Another useful thing to do is go through `E-needs-mcve` and `E-needs-bisection` issues and creating minimizations or bisecting the issue
+(using [cargo-bisect-rustc](`https://github.com/rust-lang/cargo-bisect-rustc`)). When you provide one, you can also remove the label
+using rustbot (`@rustbot label -E-needs-bisection`).
+
+## Labels
+
+There are many different labels that can be applied to issues.
+
+- `needs-triage`: signals that an issue is new and needs initial triage
+- `T-*`: Specifies the team or teams that this issue is relevant to, for example compiler, types or libs
+- `C-*`: Specifies the category of the label, for example a bug, tracking issue or discussion
+- `O-*`: For platform-specific issues, specifies the platform (architecture or operating system). For example macos, aarch64, windows
+- `A-*`: The areas that the issue is relevant to, for example linkage, patterns, diagnostics
+- `F-*`: When the issue concerns a specific (usually unstable) feature
+- `requires-nightly`: This issue is not relevant to the stable compiler
+- `requires-{incomplete,internal}-features`: This issue requires an incomplete or internal feature. The latter often means that the issue
+    should be closed in accordance with compiler [MCP 620](https://github.com/rust-lang/compiler-team/issues/620).
+- `regression-*`: Labels for tracking issues that are regressions.
+- `D-*`: Labels for diagnostics issue.
+- `I-*`: Different labels about the nature (originally, importance) of a bug. For example ICE, slow code, heavy code, crashes, unsoundness.
+- `P-*`: Priority labels. Applied using the [Compiler Prioritization procedure](../compiler/prioritization.md)
+- `S-*`: The status of an issue, for example S-needs-repro.
+- `E-*`: Calls for participation, for example to minimize an issue (E-needs-mcve) or because there is mentoring available (E-mentor).
```

Comment:
Should we call out <kbd>E-{easy,medium,hard}</kbd> separately here mentioning that you should only add them if you are an expert in the given area of the compiler etc. and know for certain how difficult it would be for someone else to fix it? Maybe that's obvious though

Reply:
I feel that the compiler experts often evaluate things as easier than they actually are due to the [curse of knowledge](https://en.wikipedia.org/wiki/Curse_of_knowledge). Experts are also not immune to being ambushed by yak-shaving problems: I have seen things be evaluated as _relatively_ simple by an expert, and the issue actually happens to involve at least one more yak than expected. These labels feel inherently dubious to me.

Reply:
I agree; if anything we should label issues as mentored and separately advocate for experts to mentor issues. That has a much better chance of resulting a smooth contributor experience than "this seems easy"